src/main/java/org/broadinstitute/consent/http/service/UseRestrictionConverter.java

```diff
@@ -124,77 +38,95 @@ public DataUse parseDataUsePurpose(DataAccessRequest dar) {
       //
       //    Research related entries
       //
-      if (Objects.nonNull(dar.getData().getMethods())) {
+      if (Objects.nonNull(dar.getData().getMethods()) && Boolean.TRUE.equals(dar.getData().getMethods())) {
         dataUse.setMethodsResearch(dar.getData().getMethods());
       }
-      if (Objects.nonNull(dar.getData().getPopulation())) {
-        dataUse.setPopulationStructure(dar.getData().getPopulation());
+      if (Objects.nonNull(dar.getData().getPopulation()) && Boolean.TRUE.equals(dar.getData().getPopulation())) {
         dataUse.setPopulation(dar.getData().getPopulation());
       }
-      if (Objects.nonNull(dar.getData().getControls())) {
-        dataUse.setControlSetOption("Yes");
+      if (Objects.nonNull(dar.getData().getControls()) && Boolean.TRUE.equals(dar.getData().getControls())) {
         dataUse.setControls(dar.getData().getControls());
       }
 
       //
       //    Diseases related entries
       //
 
-      List<String> ontologies = dar.getData().getOntologies()
-          .stream().map(OntologyEntry::getId).collect(Collectors.toList());
+      List<String> ontologies = dar.getData()
+          .getOntologies()
+          .stream()
+          .map(OntologyEntry::getId)
+          .toList();
       if (CollectionUtils.isNotEmpty(ontologies)) {
         dataUse.setDiseaseRestrictions(ontologies);
       }
 
-      //
-      //    gender, age and commercial status entries
-      //
+      // commercial status
       if (Objects.nonNull(dar.getData().getForProfit())) {
-        dataUse.setCommercialUse(dar.getData().getForProfit());
+        if (Boolean.TRUE.equals(dar.getData().getForProfit())) {
+          dataUse.setNonProfitUse(!dar.getData().getForProfit());
+          dataUse.setCommercialUse(dar.getData().getForProfit());
+        } else {
+          dataUse.setNonProfitUse(dar.getData().getForProfit());
+          dataUse.setCommercialUse(!dar.getData().getForProfit());
+        }
```

Comment:
```suggestion
        Boolean isForProfit = Boolean.TRUE.equals(dar.getData().getForProfit());
        dataUse.setCommercialUse(isForProfit);
        dataUse.setNonProfitUse(!isForProfit);
```

nit: simplify these calls